src/nvim/strings.c

```diff
@@ -45,6 +45,41 @@
 #include "nvim/os/os.h"
 #include "nvim/os/shell.h"
 
+/// Compares two strings for equality.
+bool strequal(const char *a, const char *b)
+  FUNC_ATTR_NONNULL_ALL FUNC_ATTR_PURE
+{
+  return strcmp(a, b) == 0;
+}
+
+/// Compares two sized strings for equality.
+bool strnequal(const char *a, const char *b, size_t size)
+  FUNC_ATTR_NONNULL_ALL FUNC_ATTR_PURE
+{
+  return strncmp(a, b, size) == 0;
+}
+
+/// Compares two strings for equality ignoring differences in case.
```

Comment:
> $ man 3 strcasecmp actually says for me: "compare two strings ignoring case".

I didn't mean to imply otherwise; it was a suggestion I thought up independently. Anyways, up to you as they're both clear.